crates/fiber-lib/src/fiber/channel.rs

```diff
@@ -2196,6 +2204,59 @@ where
                 }
                 Ok(())
             }
+            ChannelCommand::CheckShutdown => {
+                // check channel ready state
+                if state.state == ChannelState::ChannelReady {
+                    // check shutdown transactions
+                    let request = GetShutdownTxRequest {
+                        funding_lock_script: state.get_funding_lock_script(),
+                    };
+                    match call!(self.network, |tx| NetworkActorMessage::Command(
```

Comment:
I think this cron job is better moved into Network actor, `call!` is a bad smell for me, since we met deadlock bug because of channel actor send message to network actor and waiting for reply, and at the same time network actor send message to channel actor and waiting for reply. seems moving to network actor could avoid use `call!`.